cffi/cdefs.h

```diff
@@ -434,6 +437,21 @@ struct lyd_node_leaf_list {
 	...;
 };
 
+typedef enum {
+    LYD_DIFF_END = 0,
```

Comment:
Instead of specifying the value here, you should use the `...` ellipsis after the last `enum` value you want to export to let CFFI use the values from `libyang.h`. That way, if they change the values, the extension will follow automatically.